rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java

```diff
@@ -86,6 +88,15 @@
 
     protected long counter = 0;
```

Comment:
should be an atomicLong?

Reply:
I thing it worked due to single thread context it was used - the class it self is Active Object.
I do not know if it better to change it to AtomicLong. Will ask Justine